test/Test/Toml/Parsing/Unit.hs

```diff
@@ -161,15 +162,15 @@ spec_Parser = do
                 $ do
                       parseInteger "-9223372036854775808" (-9223372036854775808)
                       parseInteger "9223372036854775807"  9223372036854775807
-          --xit "can parse numbers with underscores between digits" $ do
-          --  parseInt "1_000" 1000
-          --  parseInt "5_349_221" 5349221
-          --  parseInt "1_2_3_4_5" 12345
-          --  parseInt "1_2_3_" 1
-          --  parseInt "13_" 13
-          --  intFailOn "_123_"
-          --  intFailOn "_13"
-          --  intFailOn "_"
+            it "can parse numbers with underscores between digits" $ do
+               parseInteger "1_000" 1000
+               parseInteger "5_349_221" 5349221
+               parseInteger "1_2_3_4_5" 12345
+               parseInteger "1_2_3_" 1
+               parseInteger "13_" 13
```

Comment:
That's great that you found these tests and uncommented them! These tests should be enough.